sns-book/src/domain-name/domain-find-for-owner.md

```diff
@@ -1,6 +1,28 @@
 # Get all domains of a user
 
-You can retrieve all the domain names of a user with the following `MemcmpFilter`
+If you are using `@bonfida/spl-name-service` you can use the following code:
+
+```js
+import { getAllDomains } from "@bonfida/spl-name-service";
+
+// ...
+
+const domains = await getAllDomains(connection, user);
+```
+
+The function above will return an array of 2 Public Keys. Use the toBase58() method to convert the keys into a human readable format.
```

Comment:
1. Why did you write `2 Public Keys`? I think it's just an array of public keys
2. `Use the toBase58() method to convert the keys into a human readable format` human readable reads a bit weird to me in this context as it usually refers to reverses. `Use the toBase58() method to convert public keys to a base 58 encoded strings` seems a bit more accurate

Reply:
1.) This is a typo, I'll update
2.) I'll update this as well, your description describes the method better